src/Database/Schema/TableSchemaInterface.php

```diff
@@ -78,6 +78,13 @@ interface TableSchemaInterface extends SchemaInterface
      */
     const TYPE_STRING = 'string';
 
+    /**
+     * String column type
+     *
+     * @var string
+     */
+    const TYPE_CHAR = 'char';
```

Comment:
This new type doesn't have a TypeClass or a mapping to make it the same as `string`. The `char` type will also need SQL generation code in all of the database adapter layers or fixture schema will not be generated using `char` columns when the fixture has `type => char`.

Reply:
Oh damn! My bad. Sorry, yea should have checked that bit. It would result in breaking as 'Unknown type'.
Got it... On it...